src/jit/liveness.cpp

```diff
@@ -1323,6 +1322,13 @@ class LiveVarAnalysis
 
 void Compiler::fgLiveVarAnalysis(bool updateInternalOnly)
 {
+#if CAN_DISABLE_DFA
```

Comment:
Wow - I didn't realize there was a `CAN_DISABLE_DFA` symbol!

> I agree that we should keep this consistent with the allocator's decisions, though I think that minopts should remain a part of the condition as liveness is also used by DSE and the emitter. I think the appropriate condition under which to disable these operations is if (opts.MinOpts() && !enregisterLocalVars).

I agree about including minopts. Note, though, that `enregisterLocalVars` isn't available in liveness.